scheduler/scheduler-client/src/main/java/org/ow2/proactive/scheduler/common/util/SchedulerProxyUserInterface.java

```diff
@@ -809,6 +810,12 @@ public UserData getCurrentUserData() throws NotConnectedException {
         return uischeduler.getCurrentUserData();
     }
 
+    @Override
+    @ImmediateService
```

Comment:
Oy vey

Reply:
?

Reply:
nothing, just happy to see immediate service, but I guess it was immediate before as well